core-api/src/errors.rs

```diff
@@ -4,7 +4,6 @@ use prost_types::TimestampOutOfSystemRangeError;
 use temporal_sdk_core_protos::coresdk::{
```

Comment:
Looks like this file still has `NoWorkerForQueue` errors defined but AFAIK that's not returned anymore.